decidim-admin/spec/jobs/export_job_spec.rb

```diff
@@ -2,38 +2,37 @@
 require "spec_helper"
 
 module Decidim
-  module Proposals
+  module Admin
     describe ExportJob do
```

Comment:
No need to do that, it's an internal. All a feature has to do is declare a collection and serializer via its DSL!